bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/constants/LinkyBindingConstants.java

```diff
@@ -0,0 +1,136 @@
+    public static final String LINKY_REMOTE_MONTHLY_GROUP = "monthly";
+    public static final String LINKY_REMOTE_YEARLY_GROUP = "yearly";
+    public static final String LINKY_TEMPO_CALENDAR_GROUP = "tempo-calendar";
+    public static final String LINKY_REMOTE_LOAD_CURVE_GROUP = "load-curve";
+
+    // List of all Channel id's
+    public static final String CHANNEL_CONSUMPTION = "consumption";
+    public static final String CHANNEL_MAX_POWER = "max-power";
+    public static final String CHANNEL_POWER = "power";
+    public static final String CHANNEL_TIMESTAMP_CHANNEL = "power";
+
+    public static final String CHANNEL_DAY_MINUS_1 = "yesterday";
+    public static final String CHANNEL_DAY_MINUS_2 = "day-2";
+    public static final String CHANNEL_DAY_MINUS_3 = "day-3";
+
+    public static final String CHANNEL_PEAK_POWER_DAY_MINUS_1 = "power";
+    public static final String CHANNEL_PEAK_POWER_TS_DAY_MINUS_1 = "timestamp";
+
+    public static final String CHANNEL_PEAK_POWER_DAY_MINUS_2 = "power-2";
+    public static final String CHANNEL_PEAK_POWER_TS_DAY_MINUS_2 = "timestamp-2";
+
+    public static final String CHANNEL_PEAK_POWER_DAY_MINUS_3 = "power-3";
+    public static final String CHANNEL_PEAK_POWER_TS_DAY_MINUS_3 = "timestamp-3";
+
+    public static final String CHANNEL_WEEK_MINUS_0 = "thisWeek";
+    public static final String CHANNEL_WEEK_MINUS_1 = "lastWeek";
+    public static final String CHANNEL_WEEK_MINUS_2 = "week-2";
+
+    public static final String CHANNEL_MONTH_MINUS_0 = "thisMonth";
+    public static final String CHANNEL_MONTH_MINUS_1 = "lastMonth";
+    public static final String CHANNEL_MONTH_MINUS_2 = "month-2";
+
+    public static final String CHANNEL_YEAR_MINUS_0 = "thisYear";
+    public static final String CHANNEL_YEAR_MINUS_1 = "lastYear";
+    public static final String CHANNEL_YEAR_MINUS_2 = "year-2";
+
+    public static final String CHANNEL_TEMPO_TODAY_INFO = "tempo-info-today";
+    public static final String CHANNEL_TEMPO_TOMORROW_INFO = "tempo-info-tomorrow";
+    public static final String CHANNEL_TEMPO_TEMPO_INFO_TIME_SERIES = "tempo-info-timeseries";
+
+    public static final String PROPERTY_IDENTITY = "identity";
+
+    public static final String PROPERTY_CONTRACT_LAST_ACTIVATION_DATE = "contractLastActivationdate";
+    public static final String PROPERTY_CONTRACT_DISTRIBUTION_TARIFF = "contractDistributionTariff";
+    public static final String PROPERTY_CONTRACT_OFF_PEAK_HOURS = "contractOffpeakHours";
+    public static final String PROPERTY_CONTRACT_CONTRACT_STATUS = "contractStatus";
+    public static final String PROPERTY_CONTRACT_CONTRACT_TYPE = "contractType";
+    public static final String PROPERTY_CONTRACT_LAST_DISTRIBUTION_TARIFF_CHANGE_DATE = "contractLastdistributionTariffChangedate";
+    public static final String PROPERTY_CONTRACT_SEGMENT = "contractSegment";
+
+    public static final String PROPERTY_USAGEPOINT_ID = "usagePointId";
```

Comment:
I know that I previously ask you to remove the prmId property but as now prmId is optional, I would suggest that you set the usagePointId property. It is a different name so no problem.
If possible, set it only if prmId config parameter is not set.

Reply:
Ok, it is there.